app/pages/projects/projects-search-selector.jsx

```diff
@@ -11,14 +11,12 @@ class SearchSelector extends Component {
     this.searchByName = this.searchByName.bind(this);
   }
 
-  navigateToProject(projectId) {
-    apiClient.type('projects').get(projectId)
-      .then(project => {
-        if (project.redirect != null && project.redirect.length !== 0) {
-          return browserHistory.push(project.redirect);
-        }
-        return browserHistory.push(['/projects', project.slug].join('/'));
-      });
+  navigateToProject(projectUrl) {
+    if (projectUrl.match(/^http.*/)) {
+      window.location.href = projectUrl;
```

Comment:
I think this should be `window.location = projectUrl;` or `window.location.assign(projectUrl);` as per the first example at https://developer.mozilla.org/en-US/docs/Web/API/Window/location